src/manifest/manifest.ts

```diff
@@ -720,17 +729,19 @@ export default class Manifest extends EventEmitter<IManifestEvents> {
 }
 
 /**
- * Update decipherability based on a predicate given.
- * Do nothing for a Representation when the predicate returns false, mark as
- * undecipherable when the predicate returns false. Returns every updates in
- * an array.
+ * Update `decipherable` property of every `Representation` found in the
+ * Manifest based on the result of a `isDecipherable` callback:
+ *   - When that callback returns `true`, update `decipherable` to `true`
+ *   - When that callback returns `false`, update `decipherable` to `false`
+ *   - When that callback returns `undefined`, update `decipherable` to
+ *     `undefined`
  * @param {Manifest} manifest
- * @param {Function} predicate
+ * @param {Function} isDecipherable
  * @returns {Array.<Object>}
  */
 function updateDeciperability(
   manifest : Manifest,
-  predicate : (rep : Representation) => boolean
+  isDecipherable : (rep : Representation) => boolean | undefined
```

Comment:
I found that updated function much easier to understand than the previous one.
So even if this PR is not merged, this is something I would like to update anyway.